### DISCUSSION TITLE
Moving rlgl to a separate repo

### DISCUSSION THREAD

**Discussion by Germ210** [raysan5/raylib#4256] [Ideas]
Hello, I just thought it would be a good idea to move rlgl to it's own separate repository. Since, rlgl is pretty useful on its own and you can link it with angle easier, I guess

**Comment by Peter0x44** [comment]
Raylib depends on rlgl so I don't think this is viable.

**Reply by Germ210** [reply]
I should be more clear: yes rlgl would still be in raylib, but also as a standalone library in it's own repo, with different headers for different versions. If not, I don't mind doing it myself

**Reply by Peter0x44** [reply]
Then changes/PRs to it would have to be done twice?
You can use curl or wget or something to download just one file from GitHub, you don't have to clone the whole repo.
https://raw.githubusercontent.com/raysan5/raylib/master/src/rlgl.h
You can also get a "raw" link to a specific commit.
Not sure of the benefits of what you're proposing.

**Reply by Germ210** [reply]
Oh ok

**Comment by raysan5** [comment]
@Germ210 No plans to move `rlgl` because it implies a separate repo to maintain and sync and I can't manage it. I already did a try with `raudio` module and it was a failure because the repo was not in sync with raylib master branch.

**Reply by Germ210** [reply]
Alright
